src/clap_app.rs

```diff
@@ -329,6 +329,24 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                 .takes_value(true)
                 .value_name("width")
                 .hidden_short_help(true)
+                .validator(
+                    |t| {
+                        let sign = t.chars().next().unwrap();
```

Comment:
Will this panic if the `t` is an empty string? Or will `clap` treat this case separately?

In any case, I think it would be a bit more clean if we used something like
``` rust
let is_offset = t.starts_with('+') || t.starts_with('-');
```
instead of checking for both `+` and `-` twice in this function.

Reply:
Done.